packages/plugin-apex/src/commands/force/apex/test/run.ts

```diff
@@ -121,6 +121,11 @@ export default class Run extends SfdxCommand {
       this.ux.warn(messages.getMessage('warningMessage'));
     }
 
+    // W-9346875 - default to human-readable result format for --wait flag
+    if (this.flags.hasOwnProperty('wait') && !this.flags.hasOwnProperty('resultformat')) {
```

Comment:
transferring over Randi's observation from the previous PR so we don't lose it - 

if the `--json` (NOT `--resultformat json`) flag is passed we should be displaying results in that format rather than a default human-readable format.